Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM navikt/java:16
+FROM navikt/java:17
```

Comment:
Her har vi oppskrift på å fase ut navikt sine baseimages. Skal vi kanskje gjøre det i samme slengen?

Reply:
https://nav-it.slack.com/archives/C01BSCJM127/p1667550844151729

Reply:
Tenkte og ikke blande det med azure ad tingene, men om vi deployer det alene først og ser det fuker fint så burde det være greit. Ser litt på det da .